driver-reactive-streams/src/test/unit/com/mongodb/reactivestreams/client/internal/FindPublisherImplTest.java

```diff
@@ -88,7 +88,7 @@ void shouldBuildTheExpectedOperation() {
                 .retryReads(true)
                 .filter(new BsonDocument())
                 .allowDiskUse(false)
-                .batchSize(100)
+                .batchSize(101)
```

Comment:
I suspect that if you move the logic to FindOperation, you can revert the test changes.